### DISCUSSION TITLE
How to change the date format to a different one ?

### DISCUSSION THREAD

**Discussion by Mysterius** [filebrowser/filebrowser#2746] [Q&A]
Hello,

When using "exact date format", the "last modified" column shows the date in the "MM/DD/YYY" format and the time as a 12-hours clock.

How can I change the date to another format ? (specifically DD/MM/YYY and 24 hours clock) 

My system is running in my national format, so I figured Filebrowser doesn't pull the date format from the system.

Thank you !

**Comment by ryjogo** [comment]
dupe of #1270  ?

**Reply by Mysterius** [reply]
Not a duplicate, no. I'm talking about adding a setting to change the display of the exact date format.
The USA are the only country using the MM/DD/YYYY format. Almost everyone else uses a variation of DD/MM/YYYY and it's extremely misleading to some users.

**Comment by Nerahikada** [comment]
Fixed by #2952 to display the time notation according to the user's locale.
